models/query_tables_accessed.sql

```diff
@@ -0,0 +1,41 @@
+
+{{ config(
+    materialized='incremental',
+    unique_key=['_unique_key', 'query_start_time'],
+) }}
+
+with
+access_history as (
+    select *
+    from {{ ref('stg_access_history') }}
```

Comment:
Should we get `OBJECTS_MODIFIED` while we're at it? Or do you think that should be a different table?

Reply:
will have to be a different table